Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,228 @@
+  (c) An organization may have another organization as part.
+  True. Organizations break off separate organizations to perform different functions all of the time. A charity organization for finding housing for the homeless might break off local and regional organizations to better serve people at various locations for example.
+  (d) An organization may have no members as part. 
+  True. Organizations can continue their existence even after losing all members for a time. For example, the Soviet Union continued to exist for a time even after all member states left the organization. It's continuation, then, depended not on its members but instead on other factors like the organizational structure and rules.
+  (e) Any site is partially bounded by some instance of Material Entity.
+  False. It is possible for a site to only be a part of an immaterial entity without being itself bounded by a material entity.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  False. The book may have been primarily used to read or write in, but it was always a function of it that it could take up space and act as a solid base for another object. Function is not limited by its purpose of creation, and instead includes its ability to hold up a table or any other such object by virtue of its physical make-up.
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break.
+  True. Being cushioned with packing tape does not change the dispositions of the vase itself. 
+  (h) Spacetime is a class in BFO.
+  False. The spatiotemporal region covers this, but this itself is not a class.
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+  False. We could construct any number of subclasses of zero-, one-, or two-dimensional continuant fiat boundaries, which would be subclasses of continuant fiat boundary.
+```
+
+8. Model the following scenario in BFO, introducing whatever terms are needed to do so: John runs for 3 hours, startin slowly, speeding up during the middle, then ending the run at a slower pace. 
+
+John is an instance_of object
+John's Running is an instance_of process
+John participates in John's Running
+John bears running speeds of slow, normal, and fast
+Slow, normal, and fast running speeds are instances_of dispositions
+John’s running occurs in 3-hours, which is instance_of one-dimensional temporal region.
+John's running consists of three temporal parts: Beginning, Middle, and End
+Beginning, Middle, and End are instances_of one-dimensional temporal regions
+Beginning realizes normal running speed
```

Comment:
Processes can't realize dispositions. On your model, John bears, say, normal running speed disposition that is realized in a process of normal running speed and that process is temporal part of John's running which projects onto a proper part of the 1-D temporal region on which John's running projects